tests/IntegrationTests/ItemPropertiesSetupTest.php

```diff
@@ -101,22 +117,47 @@ public function shouldExecuteEventWithJustCreatedProperties(): void
      */
     public function shouldDeleteCreatedPropertiesFromMatej(): void
     {
-        $response = $this->createMatejInstance()
-            ->request()
-            ->deleteItemProperties()
-            ->addProperty(Command\ItemPropertySetup::boolean('integration_test_php_client_bool'))
-            ->addProperty(Command\ItemPropertySetup::double('integration_test_php_client_double'))
-            ->addProperty(Command\ItemPropertySetup::int('integration_test_php_client_int'))
-            ->addProperty(Command\ItemPropertySetup::string('integration_test_php_client_string'))
-            ->addProperties([
-                Command\ItemPropertySetup::timestamp('integration_test_php_client_timestamp'),
-                Command\ItemPropertySetup::set('integration_test_php_client_set'),
-            ])
-            ->send();
+        $request = $this->fillPropertyRequestBuilder(
+            $this->createMatejInstance()->request()->deleteItemProperties(),
+            $this->provideProperties()
+        );
 
+        $response = $request->send();
         $this->assertSame(6, $response->getNumberOfCommands());
         $this->assertSame(6, $response->getNumberOfSuccessfulCommands());
         $this->assertSame(0, $response->getNumberOfFailedCommands());
         $this->assertSame(0, $response->getNumberOfSkippedCommands());
     }
+
+    /**
+     * Fill the request builder with given properties
+     *
+     * @param string[][] $properties type => name
+     */
+    private function fillPropertyRequestBuilder(
+        ItemPropertiesModifierRequestBuilder $builder,
+        array $properties
+    ): ItemPropertiesModifierRequestBuilder {
+        foreach ($properties as $name => $definition) {
+            $property = forward_static_call([Command\ItemPropertySetup::class, $definition[0]], $name);
```

Comment:
`ItemPropertySetup::$definition[0]($name)` :)?

Reply:
That wouldn't work as PHP would think that $definition is a static array on `ItemPropertySetup` class. Would have to be `ItemPropertySetup::{$definition[0]}($name)` which I found pretty ugly :|